contracts/queue/task.go

```diff
@@ -1,16 +1,16 @@
 package queue
 
 import (
-	"time"
+	"github.com/goravel/framework/support/carbon"
 )
 
 type Task interface {
 	// Dispatch dispatches the task.
 	Dispatch() error
 	// DispatchSync dispatches the task synchronously.
 	DispatchSync() error
-	// Delay dispatches the task after the given delay.
-	Delay(time time.Time) Task
+	// Delay dispatches the task after the given time.
+	Delay(time carbon.Carbon) Task
```

Comment:
Adjusted to use Carbon objects, because Laravel does this too, see https://learnku.com/docs/laravel/10.x/queues/14873#38a860

Reply:
Good idea.